packages/@yoda/audio/index.js

```diff
@@ -174,6 +175,10 @@ AudioManager.setVolume = function (type, vol) {
   }
 
   var stream = AudioBase[type]
+
+  process.nextTick(() => {
+    floraDisposable.post(`rokid.audio.volume-changed`, [ stream.name, vol ])
```

Comment:
No necessary to use string template here, BTW the name should be `yodart.audio.onvolumechange` would make sense.